Firestore/core/test/unit/remote/serializer_test.cc

```diff
@@ -1512,6 +1513,62 @@ TEST_F(SerializerTest, EncodesResumeTokens) {
   ExpectRoundTrip(model, proto);
 }
 
+TEST_F(SerializerTest, EncodesExpectedCount) {
+  core::Query q = Query("docs");
+  TargetData model(q.ToTarget(), 1, 0, QueryPurpose::Listen,
+                   SnapshotVersion::None(), SnapshotVersion::None(),
+                   Bytes({1, 2, 3}), /*expected_count=*/2);
```

Comment:
Can you use a more "distinct" value than `2` for `expected_count`? Maybe 1234 or 42? It just makes the test a bit easier to read because you can look around for that "magic" value. Also on line 1551.